Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -1484,6 +1493,397 @@ def gsuite_chromeos_device_action_command(client: Client, args: dict[str, str])
+    return CommandResults(outputs_prefix=OUTPUT_PREFIX['CHROME_BROWSERS'],
+                          outputs_key_field=['deviceId'],
+                          outputs=cb_list_resp,
+                          readable_output=readable_output,
+                          raw_response=response)
+
+
+def modify_policy_command(client: Client, args: dict[str, str]) -> str:
+    """
+        get a user details based on user key.
+
+        :param client: Client object.
+        :param args: Command arguments.
+
+        :return: String that confirms request was executed.
+    """
+    client.set_authorized_http(scopes=SCOPES['POLICY_MANAGEMENT'])
+    customer_id = args.get('customer_id', '')
+    target_type = args.get('target_type', '')
+    policy_raw_json = args.get('policy_raw_json', '')
+    policy_field_json_entry_id = args.get('policy_field_json_entry_id', '')
+    target_resource = args.get('target_resource', '')
+    additional_target_keys = args.get('additional_target_keys', '')
+    policy_schema = args.get('policy_schema', '')
+    policy_value = args.get('policy_value', '')
+    update_mask = args.get('update_mask', '')
+
+    if target_type == 'Group':
+        full_url = f'https://chromepolicy.googleapis.com/v1/customers/{customer_id}/policies/groups:batchModify'
+        target_resource_customized = f'groups/{target_resource}'
+    else:
+        full_url = f'https://chromepolicy.googleapis.com/v1/customers/{customer_id}/policies/orgunits:batchModify'
+        target_resource_customized = f'orgunits/{target_resource}'
+
+    atk_temp = '{' + additional_target_keys + '}'
+    atk_dict = json.loads(atk_temp)
+
+    if policy_raw_json:
+        app_payload = GSuiteClient.safe_load_non_strict_json(policy_raw_json)
+    elif policy_field_json_entry_id:
+        app_payload = safe_load_json(policy_field_json_entry_id)
+    else:
+        app_payload = {
+            "requests": [
+                {
+                    "policyTargetKey":
+                    {
+                        "targetResource": target_resource_customized,
+                        "additionalTargetKeys": atk_dict
+                    },
+                    "policyValue": {
+                        "policySchema": policy_schema,
+                        "value": {
+                            "appInstallType": policy_value
+                        }
+                    },
+                    "updateMask": update_mask
+                }
+            ]
+        }
+
+    client.http_request(full_url=full_url, method='POST', body=app_payload)
+
+    # Output
+    return f'Policy has been modified for the customer {customer_id}'
+
+
+def policy_resolve_command(client: Client, args: dict[str, str]) -> CommandResults:
+    """
+        resolve the provided policy and return its details
+
+        :param client: Client object.
+        :param args: Command arguments.
+
+        :return: Command Result.
+    """
+    API_LIMIT = 1000
```

Comment:
same notes here as above regarding the pagination implementation (won't repeat this comment, but same for the next ones)